### PR TITLE
candidate consent management

### PR DESCRIPTION
**Pull request by robeecaneda1**

### CODE REVIEW

File: Bullhorn/Candidate-Consent.md [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-
-![Candidate Status "Do Not Use"](img/Bullhorn-Candidate-GDPR-DoNotUseStatus.png)
-
-This status does not automatically mean that a mass mailing will not be sent to these candidates.
-But we can filter out all the Candidates with this status before sending a mass mail.
-
-Consent (GDPR)
---------------
-
-![Candidate Consent for GDPR](img/Candidate-Consent-Tab.png)
```
Comment (Laoujin) [inline Bullhorn/Candidate-Consent.md]: Delete the image and add it to the PR

File: Bullhorn/Candidate-Consent.md [hunk 2]
```diff
@@ -1,20 +0,0 @@
-Candidate Consent
-=================
-
-Candidate > Edit > Status:
-
-There is a specific status "DO NOT USE".
-
-![Candidate Status "Do Not Use"](img/Bullhorn-Candidate-GDPR-DoNotUseStatus.png)
```
Comment (Laoujin) [inline Bullhorn/Candidate-Consent.md]: Delete the image.
If useful, add a new Zoho image

File: Bullhorn/Candidate-Consent.md (hunk 1 above)
Comment (robeecaneda1) [reply Bullhorn/Candidate-Consent.md]: deleted the Bullhorn image for the guide

File: Bullhorn/Candidate-Consent.md (hunk 2 above)
Comment (Laoujin) [reply Bullhorn/Candidate-Consent.md]: *bump*

Comment (robeecaneda1) [reply Bullhorn/Candidate-Consent.md]: image and guide has been deleted.

**Pull request merged**